src/python/env.py

```diff
@@ -247,7 +267,15 @@ def step(  # pyright: ignore[reportIncompatibleMethodOverride]
         # Frameskip
         reward = 0.0
         for _ in range(frameskip):
-            reward += self.ale.act(self._action_set[action])
+            if self.continuous:
+                action = tuple(action)
+                if len(action) != 3:
+                    raise error.Error("Actions must have 3-dimensions.")
+
```

Comment:
I think this is not needed, Gymnasium's passive env checker automatically checks for this and line 275 will error out anyway. @pseudo-rnd-thoughts could you confirm?